thorlcr/graph/thgraph.cpp

```diff
@@ -2462,12 +2461,14 @@ void CJobBase::startJob()
     }
     PrintMemoryStatusLog();
     logDiskSpace();
-    unsigned keyNodeCacheMB = (unsigned)getWorkUnitValueInt("keyNodeCacheMB", 0);
-    if (keyNodeCacheMB)
```

Comment:
it meant that if "keyNdoeCacheMB" was not set, the above returned 0 and setNodeCacheMem was never called.

Now it asserts a default value and always calls.